app/views/health_cards/show.pdf.erb

```diff
@@ -87,7 +87,9 @@
         <% @qr_codes.chunks.each do |ch| %>
           <div class="tile is-parent">
             <div class="tile is-child box has-text-centered">
-              <%= wicked_pdf_image_tag patient_qr_code_url(@patient, ch.ordinal, format: :png) %>
+              <%= wicked_pdf_image_tag url_for(controller: 'qr_codes', action: 'show', host: "localhost",
+                                               protocol: 'http', port: ENV['PORT'], patient_id: @patient.id,
```

Comment:
It looks like this causes the QR Code in the PDF to not render when running using Docker (it still works fine when running with docker-compose).

I was able to fix this by adding `ENV PORT=3000` to `Dockerfile`.

Reply:
Good catch. Fixed in https://github.com/dvci/health_cards/pull/97/commits/42ad67f0a9ee6791cc17a9194ec458ecbd3f6349.

Reply:
QR Code properly renders now!